tools/print-all/print-all

```diff
@@ -106,8 +106,8 @@ for fil in sh.git('ls-files', _iter=True):
     try:
         filetype = extensions[os.path.basename(fil).split('.')[1]]
     except (KeyError, IndexError):
-        with open(fil, 'rb') as f:
-            header = f.readline()
+        with open(fil, 'rb') as file:
+            header = file.readline()
```

Comment:
In a context manager such as  `with <expr> as var`, `var` lives on after clean-up. This is an issue as the type of a name can only be assigned once. So if I have:
```
with open('test.txt', 'r') as f:  # f is TextIO
    ...
with open('data.dat', 'rb') as f: # error, expression evaluates to BytesIO, but f is TextIO
    ...
```
This will raise a type error. 

I will change it to a different more descriptive name, I haven't written Python 2 in so long I forgot `file` was a keyword.